FreeAPS/Sources/Modules/AddCarbs/View/AddCarbsRootView.swift

```diff
@@ -49,7 +49,7 @@ extension AddCarbs {
                             autofocus: true,
                             cleanInput: true
                         )
-                        Text("grams").foregroundColor(.secondary)
+                        Text(state.carbs > state.maxCarbs ? "⚠️" : "g").foregroundColor(.secondary)
```

Comment:
the `g` here is actually a localized value --> `NSLocalizedString(" g", comment: "gram of carbs")`

not sure if this should or shouldn't be hard coded here.